docs/tutorial/part-eight/index.md

```diff
@@ -101,33 +101,33 @@ npm install --save gatsby-plugin-manifest
         start_url: `/`,
         background_color: `#6b37bf`,
         theme_color: `#6b37bf`,
-        // Enables "Add to Homescreen" prompt and disables browser UI (including back button)
-        // see https://developers.google.com/web/fundamentals/web-app-manifest/#display
+        // Ermöglicht das "Zum Homebildschirm hinzufügen" und deaktiviert die Browser UI (inkl. Zurück-Button)
+        // weitere Infos: https://developers.google.com/web/fundamentals/web-app-manifest/#display
         display: `standalone`,
-        icon: `src/images/icon.png`, // This path is relative to the root of the site.
+        icon: `src/images/icon.png`, // Dies ist ein relativer Pfad zum Root deines Projekts.
       },
     },
   ]
 }
 ```
 
-That's all you need to get started with adding a web manifest to a Gatsby site. The example given reflects a base configuration -- Check out the [plugin reference](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode) for more options.
+Das ist alles, was du brauchst, um ein Web manifest zu deinem Gatsby Projekt hinzuzufügen. Es handelt sich hierbei um die Grundkonfiguration. Für weitere Einstellungsmöglichkeiten, schau in die [Plugin Dokumentation](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode).
 
-## Add offline support
+## Mach dein Projekt offline verfügbar
 
-Another requirement for a website to qualify as a PWA is the use of a [service worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API). A service worker runs in the background, deciding to serve network or cached content based on connectivity, allowing for a seamless, managed offline experience.
+Eine weitere Voraussetzung dafür, dass dein Projekt als PWA erkannt wird, ist das Nutzen eines sog. [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API). Ein Service Worker läuft im Hintergrund und entscheidet, ob er Inhalte aus dem Cache oder aus dem Netzwerk anzeigen soll. Dadurch funktioniert deine Seite unabhängig davon, ob dein Client im Internet verbunden ist, oder nicht.
```

Comment:
```suggestion
Eine weitere Voraussetzung dafür, dass dein Projekt als PWA erkannt wird, ist das Nutzen eines sog. [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API). Ein Service Worker läuft im Hintergrund und entscheidet bspw., ob er Inhalte aus dem Cache oder aus dem Netzwerk anzeigen soll. Dadurch funktioniert deine Seite unabhängig davon, ob dein Client im Internet verbunden ist, oder nicht.
```

Because a service worker can do more than that :)